http/src/main/resources/org/broadinstitute/dsde/workbench/leonardo/liquibase/changesets/20230927_add_cluster_table_index_on_internal_id.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog logicalFilePath="leonardo" xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd">
+
+    <changeSet logicalFilePath="leonardo" author="mnolting" id="add_cluster_table_index_on_internal_id">
+        <addUniqueConstraint tableName="CLUSTER" columnNames="internalId" constraintName="IDX_CLUSTER_INTERNAL_ID" />
```

Comment:
This might not work in pre-existing Leo databases.. we used to name the Sam resources like `{projectName}_{runtimeName}`, which means the sam resource id could be reused if a runtime was created with the same name. For example here's dev:
```
MySQL [leonardo]> select internalId, count(1) FROM CLUSTER group by 1 order by 2 desc limit 5;
+------------------------------------+----------+
| internalId                         | count(1) |
+------------------------------------+----------+
| aou-rw-test-137-2_all-of-us        |       38 |
| aou-rw-test-141-2_all-of-us        |       37 |
| aou-rw-local1-1557534645_all-of-us |       29 |
| aou-rw-local1-1766345006_all-of-us |       25 |
| aou-rw-local1-928015957_all-of-us  |       24 |
+------------------------------------+----------+
```

Maybe this should be an index without a unique constraint.

Reply:
yes, that would work! can  make nonunique.

Reply:
Done!